.github/workflows/unit-test.yml

```
@@ -38,5 +38,28 @@ jobs:
         with:
           go-version: "1.21"
 
+      - name: Setup jq
+        uses: dcarbone/install-jq-action@v2.0.2
+        with:
+          version: |
+            type: string
+            required: false
+            description: "Version of jq to install"
+            default: "1.7"
+
+      - name: Check jq
+        run: |
+          which jq
+          jq --version
+
       - name: run unit tests
-        run: GOPATH=$(go env GOPATH) make -j $(nproc) test
+        run: |
+          export ROOK_UNIT_JQ_PATH="$(which jq)"
+          GOPATH=$(go env GOPATH) make -j $(nproc) test | tee output.txt
+          if grep "skipping mds liveness probe script unit tests because jq binary location is not known" output.txt
+          then
+              echo "jq not found, fail the test"
+              rm output.txt
+              exit 1
+          fi
+          rm output.txt
```

Why remove the output file? I don't think it's necessary to clean up the runner.

```suggestion
          if grep "skipping mds liveness probe script unit tests because jq binary location is not known" output.txt
          then
              echo "jq not found, fail the test"
              exit 1
          fi
```

---

We do this at other places with files and resources we create so I think it is a good and common practice,
But yaa not necessary